### PR TITLE
(Update) Laravel 6 Support

### PR DESCRIPTION
**Pull request by HDVinnie**
- add Laravel 6 support

### CODE REVIEW

**Comment by HDVinnie** [comment]
@christofferok when you have a chance please. Trying to update some of my apps.

**Comment by HDVinnie** [comment]
- ref #16

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": "~5.6|~7.0",
         "emojione/emojione": "^4.0.0",
-        "illuminate/support": "~5.1"
+        "illuminate/support": "5.8.*|6.0.*"
```
Comment (christofferok) [inline composer.json]: Now that Laravel has moved to semantic versioning, wouldn't `~6.0` be better? Then it will work with new 6.x releases also. And maybe keep `~5.1` as well? No need to cut support for them?
Do you think `~5.1|~6.0` would be sufficient?

File: composer.json [hunk 2]
```diff
@@ -20,9 +20,9 @@
         }
     ],
     "require": {
-        "php": "~5.6|~7.0",
+        "php": ">=7.1",
```
Comment (christofferok) [inline composer.json]: Is this change required to make it work with Laravel 6? Maybe leave it as `~5.6|~7.0` if that will let it work for users on older php versions.

**Comment by christofferok** [comment]
Thank you for the PR @HDVinnie 👍🏼 
Left a few comments/questions for some of the changes.

**Comment by HDVinnie** [comment]
Updated the PR with your suggestions. You are right. No need to cut off older php versions.
